lib/mobility-core/src/Kernel/Storage/ClickhouseV2/Internal/Types.hs

```diff
@@ -153,6 +156,8 @@ showColumn (Add column1 column2) = addBrackets' (showColumn column1 <> "+" <> sh
 showColumn (CoerceNum column) = showColumn column
 showColumn (ToDate column) = "toDate" <> addBrackets' (showColumn column)
 showColumn (ToHour column) = "toHour" <> addBrackets' (showColumn column)
+showColumn (DiffUTCTime column1 column2) = "diffUTCTime" <> addBrackets' (showColumn column1 <> "-" <> showColumn column2)
+showColumn (NominalDiffTimeToSeconds column) = "nominalDiffTimeToSeconds" <> addBrackets' (showColumn column)
```

Comment:
did not found these functions `diffUTCTime`,  `nominalDiffTimeToSeconds` in clickhouse docs, are you sure they are correct?

Reply:
also second function `nominalDiffTimeToSeconds` did not used in NY